From eb72c7218d2635f15423109117caadedca485736 Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Tue, 27 Jan 2026 20:23:42 +0100
Subject: [PATCH] Makefile: enable core updater & build deps/MBEDTLS

for uClibc it's advised to use HAVE_SOCKET_LEGACY for IPv4
---
 Makefile.miyoo | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/Makefile.miyoo b/Makefile.miyoo
index 3e759578ca..0dde540838 100644
--- a/Makefile.miyoo
+++ b/Makefile.miyoo
@@ -51,7 +51,8 @@ HAVE_VIDEO_FILTER = 1
 HAVE_STATIC_VIDEO_FILTERS = 1
 HAVE_STATIC_AUDIO_FILTERS = 1
 HAVE_FILTERS_BUILTIN	= 1
-HAVE_BUILTINMBEDTLS = 0
+HAVE_BUILTINMBEDTLS = 1
+HAVE_BUILTINBEARSSL = 0
 HAVE_BUILTINZLIB = 1
 HAVE_C99 = 1
 HAVE_CC = 1
@@ -82,7 +83,7 @@ HAVE_OPENGL1 = 0
 HAVE_OPENGLES = 0
 HAVE_OPENGLES3 = 0
 HAVE_OPENGL_CORE = 0
-HAVE_OPENSSL = 0
+HAVE_OPENSSL = 1
 HAVE_OVERLAY = 0
 HAVE_RBMP = 1
 HAVE_RJPEG = 1
@@ -113,6 +114,11 @@ HAVE_CORE_INFO_CACHE = 1
 #HAVE_TINYALSA = 1
 HAVE_NEAREST_RESAMPLER = 1
 HAVE_NETWORKING = 1
+HAVE_UPDATE_ASSETS = 1
+HAVE_UPDATE_CORES = 1
+HAVE_UPDATE_CORE_INFO = 1
+HAVE_SSL = 1
+HAVE_SOCKET_LEGACY = 0
 OS = Linux
 TARGET = retroarch
 
@@ -132,7 +138,7 @@ INCLUDE_DIRS = -I$(GCW0_INC_DIR)
 LIBRARY_DIRS = -L$(GCW0_LIB_DIR)
 DEFINES := -DRARCH_INTERNAL -D_FILE_OFFSET_BITS=64 -UHAVE_STATIC_DUMMY
 DEFINES += -DHAVE_C99=1 -DHAVE_CXX=1
-DEFINES += -DHAVE_GETOPT_LONG=1 -DHAVE_STRCASESTR=1 -DHAVE_DYNAMIC=1
+DEFINES += -DHAVE_GETOPT_LONG=1 -DHAVE_STRCASESTR=1 -DHAVE_DYNAMIC=1 -DHAVE_ONLINE_UPDATER -DHAVE_UPDATE_ASSETS -DHAVE_UPDATE_CORES -DHAVE_UPDATE_CORE_INFO
 DEFINES += -DHAVE_FILTERS_BUILTIN -DHAVE_ALSA
 
 SDL_DINGUX_CFLAGS := $(shell $(GCW0_SDL_CONFIG) --cflags)
-- 
2.45.2.windows.1

